### DISCUSSION TITLE
Change code highlight color

### DISCUSSION THREAD

**Discussion by ghost** [willfaught/paige#42] [Q&A] [answered]
Hi Will,

Is there a way to change to color of the code that's highlighted in [this section](https://willfaught.com/paige/shortcodes/code/#options-parameter)?

I think changing the color to something darker may make the text more readable.

Best,
Saurabh

**Comment by ghost** [comment]
Created PR #43 for improved readability.

**Reply by willfaught** [reply, marked as answer]
Let's move the discussion to your PR.
